### PR TITLE
fix ole installation + add openlayers CDN

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <link rel="stylesheet" type="text/css" href="style/style.css">
   </head>
   <body>
-    <script src="dist/ol-debug.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.4.2/ol.js"></script>
```
Comment (friedjoff) [inline index.html]: Have you you tried using the unpkg CDN? Something like https://unpkg.com/openlayers@^4/dist/ol.js would always give us the latest OpenLayers 4 version, so we can make sure OLE runs with the latest release.

File: package.json [hunk 2]
```diff
@@ -37,7 +37,8 @@
     "mustache": "^2.3.0",
     "optipng-bin": "^4.0.0",
     "pngquant": "^1.3.0",
-    "pngquant-bin": "^3.1.1"
+    "pngquant-bin": "^3.1.1",
+    "uglifyjs-webpack-plugin": "^0.4.6"
```
Comment (friedjoff) [inline package.json]: uglifyjs-webpack-plugin is already part of devDependencies so we don't need it here. Also handled in #15, so maybe just revert this change.

File: index.html (hunk 1 above)
Comment (edellucien) [reply index.html]: OK good idea, can we be sure that the updates of OL4 won't break in a way our current code? :)
If yes I just redo a new PR

Comment (friedjoff) [reply index.html]: I guess OL4 won't change to much and we should stay compatible with the current release. The index.html would be a good initial test case for that.

Comment (edellucien) [reply index.html]: OK :+1:  just made another PR
